01 - Community/People/universvm.md

```diff
@@ -0,0 +1,63 @@
+---
+aliases:
+- Leonardo Castorina
+tags:
+- 
+publish: true
+---
+
+# Leonardo Castorina
+
+- Publish sites: <https://medium.com/@universvm/> ^website
+- Twitter: [Twitter](https://twitter.com/leocastorina/) ^twitter-->
+- GitHub: [universvm](https://github.com/universvm/) ^github
+
+
+%% Feel free to add a bio below this comment %%
+
+
+## Author of
+
+%% Begin Hub: Released contributions %%
+### Guides
+- [Obsidian Tutorial for Academic Writing](https://betterhumans.pub/obsidian-tutorial-for-academic-writing-87b038060522) by [[universvm|Leonardo Castorina]]
+- [How to Boost your Productivity for Scientific Research](https://betterhumans.pub/how-to-boost-your-productivity-for-scientific-research-using-obsidian-fe85c98c63c8) by [[universvm|Leonardo Castorina]]
+
+## Sponsor this author
+- [Ko-Fi](https://ko-fi.com/leocastorina) ^ko-fi
+
+
+%% End Hub: Released contributions %%
```

Comment:
The idea behind the commented-out headings is that people un-comment the headings they have content for, and then insert their own links/content in those sections.

That way, all pages have broadly similar layout, and it is easier for us to update the pages by script in future.

It's not obvious, but the comments on lines 21 and 30 mark text which will be over-written by automated scripts.
The comment in line 32 is meant to indicate the start of manually added lists, such as the guides you added.
Please could you move the guides section down to just above the `### Others` section, which is current lines 38-40.

Also sorry if it isn't obvious, but any sponsorship links should go in the 'Sponsor this author' section which is currently commented-out on line 43, so:

1. Uncomment the 'Sponsor this author' section around the current line 43
2. Then insert your LoFi link alongside the other comment-out sponsorship links.